specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json

```diff
@@ -4477,6 +4741,10 @@
           },
           "description": "A list of references to all virtual machines in the availability set."
         },
+        "proximityPlacementGroup": {
```

Comment:
Is this property in production already and just missing in the swagger?
Based on ARM guidelines adding a property to a response is a [breaking change](https://sharepoint/sites/AzureUX/Sparta/_layouts/15/WopiFrame2.aspx?sourcedoc=%7B87E6845E-A308-44E8-ACDF-2DB17054BC77%7D&file=RestAPI_BreakingChangeGuide.docx&action=default).

Reply:
this is already in production and was added in 2018-10-01 API version, but when 2019-03-01 Swagger spec is added, this was missing